content/working_with/console/widgets/deploymentsView.md

```diff
@@ -0,0 +1,279 @@
+5. Environment Type - the value of the `csys-env-type` label for that
+   deployment
+
+   Useful for differentiating deployments that use the same blueprint.
+
+6. Location - the name of the site the deployment is assigned to
+7. The counts and statuses of child deployments depending on their type
+   (environments or services)
+
+   See [the Environments and Services section]({{< relref "#environments-and-services" >}})
+   for more information on child deployments.
+
+   The counts and statuses in those columns are based on **all** child
+   deployments (it does not matter whether they are directly or indirectly
+   attached to the main deployment).
+
+Clicking a deployment row in the table will select it and affect the contents
+of the [Details pane]({{< relref "#details-pane" >}}).
+
+At the bottom of the table are controls that allow changing the number of
+deployments shown in a single table page, as well as moving to other pages in
+case there are more deployments that fit a single page.
+
+If there is a workflow in progress for a given environment, there is a progress
+bar right under the row in the table.
+
+### Sorting deployments
+
+Deployments in the table can be sorted by clicking a column's header.
+
+Clicking the column again will cycle between ascending and descending sort
+order.
+
+## Details pane
+
+The right side of the widget contains details about the currently selected
+deployment.
+
+In the header of the right side are:
+
+- the currently selected deployment name
+- buttons to drill-down to child environments or services
+
+  The buttons will be disabled when there are no child environments or
+  services.
+
+  See [the Environments and Services section]({{< relref "#environments-and-services" >}})
+  for more information on child deployments.
+
+- buttons to execute a workflow on that deployment or invoke some other
+  deployment action
+
+What follows are different deployment-related widgets divided into 3 tabs:
+
+1. Last Execution - information about the last workflow execution for that
+   deployment
+2. Deployment Info - general information about the deployment
+3. History - a list of workflow executions for that deployment and a list of
+   logs
+
+## Environments and Services
+
+For clarity and better organization, deployments can form a hierarchical
+structure and be marked as:
+
+- environments - can hold other environments and services
+- services - the bottommost level of the hierarchy
+
+Marking the deployments happens by adding the following labels to the
+deployment:
+
+1. `csys-obj-parent` - the ID of the parent of the current deployment
+2. `csys-obj-type` - either `service` or `environment`
+   If there is no such label, the deployment is a service by default.
+
+Forming a hierarchical structure allows drilling down into deployments that are
+directly attached to a parent environment, as well as aggregating and
+propagating statuses of deployments for better problem discovery.
+
+### Deployment status
```

Comment:
I did not find any existing documentation for deployment statuses and propagating them (next section), so I described it directly here

Reply:
Good point 👍🏻 I think it would be wise to have it defined outside of the widget and reference it from here.

That's probably something we should ask @ofercloudify or @mabuaisha .

Reply:
When we will have it externalized, we can probably also update Sites Map widget in which I added such description as part of the recent Sites Map widget fix.

Ref.: https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/1832/